docs/getstarted/quick_start.md

```diff
@@ -1,119 +1,224 @@
+your own dataset. Check out
+[this article](../../guides/create_and_upload_dataset) to learn how to do that.
+
+## Running Your First Job
+
+Now that we have our project created and our dataset ready, let's run our first
+job and train our model! A [job](../getstarted/core_concepts/#jobs) is an
+execution of your code on FloydHub's deep-learning servers. To kick off a job,
+we use the `floyd run` command.
+
+Run the command below to kick off the training job. (Don't worry, we'll explain
+each piece of the command.)
 
 ```bash
-$ floyd run --gpu --env tensorflow "python mnist_cnn.py"
+$ floyd run \
+--gpu \
+--data mckay/datasets/mnist/1:/mnist \
+--env tensorflow-1.3 \
+"python mnist_cnn_train.py"
+
+Creating project run. Total upload size: 25.4KiB
 Syncing code ...
-NAME               
--------------------
-alice/quick-start/1
+[================================] 27316/27316 - 00:00:00
+
+JOB NAME
+----------------------
+alice/projects/mnist/1
 
-To view the logs enter:
-    floyd logs alice/quick-start/1
+To view logs enter:
+   floyd logs alice/projects/mnist/1
 ```
 
-Congratulations! Your first job is now running on FloydHub's GPU servers. Behind the scenes, FloydHub does the following:
+Congratulations! Your first job is now running on FloydHub's GPU servers.
+Behind the scenes, FloydHub does the following:
 
 - Syncs your local code to FloydHub's servers
 - Provisions a GPU instance on the cloud (because you set the `--gpu` flag)
-- Sets up a deep learning environment with GPU drivers and Tensorflow installed (because you set the enviroment flag to `--env tensorflow`)
+- Sets up a deep learning environment with GPU drivers and TensorFlow 1.3
+  installed (because you set the enviroment flag to `--env tensorflow-1.3`)
 - Executes the command `python mnist_cnn.py` inside this environment
 - Stores the output logs and generated output data
 - Terminates the GPU instance once the command finishes execution
 
-## Monitoring your Job
+Here is quick explanation of each part of the command you just ran:
+
+### floyd run
 
-You can view the status of your Job from your terminal using the [`floyd status`](../commands/status.md) command. You can specify a single Job name (e.g. `floyd status alice/quick-start/1`) to get its status, or the `floyd-cli` will show the status of all Jobs in the current Project.
+Tells the CLI we want to run a job.  You'll use this command to run all of your
+jobs. [More info here](../../guides/run_a_job/#instance-type).
+
+### --gpu
+
+Specifies that we want our job run on a GPU server. More info
+on instance types [here](../../guides/run_a_job/#instance-type).
+
+### --data mckay/datasets/mnist/1:/mnist
+
+The `mnist_cnn.py` script expects the MNIST data to be located at `/mnist` on
+the computer where the script runs. We can use the `--data` flag to ensure that
+our dataset is available to our code at `/mnist`.
+
+We pass the `--data` flag the name of our dataset (`mckay/datasets/mnist/1`)
```

Comment:
Probably link to the URL of the dataset somewhere so users can see the connection between the URL and the data name